.gitignore

```diff
@@ -1,25 +1,26 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
+/*.py[cod]
+/src/PIL/*.py[cod]
+/Tests/*.py[cod]
```

Comment:
Other possible directories to include here are 'winbuild' and 'docs'. I feel like there's a good argument to be made not to include winbuild, since it is only used by AppVeyor, but I would suggest including 'docs', since a user may wish to use the documentation on their local machine, yes?